src/frontend/templates/info.html

```diff
@@ -34,7 +34,35 @@ <h1>Information Page</h1>
             <p>Please sign in to view additional information.</p>
             <a href="/">Go to Main Page</a>
         {% endif %}
-        <p></p>
+        
+        <div class="info-section">
+            <h2>Introduction</h2>
+            <p>This is an LLM-powered app designed for identifying and attributing threat actors based on discovered Indicators of Compromise (IoCs).</p>
+            <h2>Features and Usage</h2>
+            <ul>
+                <p><strong>IoC Analysis:</strong> Analyze indicators like IP addresses, domains, file hashes, and other IoCs to identify potential threat actors.</p>
+                <p><strong>Conversation:</strong> .</p>
+                <p><strong>History:</strong> Review previously identified threats.</p>
+                <p><strong>Statistics:</strong> Visual summaries of threats and methods used.</p>
+            </ul>
+
+            <h2>How to Use the Application</h2>
+            <ol>
+                <p>1. Enter your IoC or more in the provided prompt.</p>
```

Comment:
I think it should say something more in line with. "Enter TTPs and potential IoCs for a given attack". The descriptions for step 2 & might need to be updated to reflect the functionality of the application.